pages/micromaster/profiles/Rudy24salazar.md

```diff
@@ -0,0 +1,24 @@
+### Rodolfo Salazar ###
+### Los Angeles, Ca ###
+### Pacific Daylight Time Zone ### 
+### MacOS Catalina 10.15.4 ###
+
+Hello everyone my name is Rodolfo Salazar but everyone calls me Rudy. I am a senior at California State University Northridge(***CSUN***).  
+  
+![C](https://arcs.center/wp-content/uploads/2019/12/New-Logo-white-outline.png)  
+
+I now live in Los Angeles, California but I was born in Guadalaja Jalisco, Mexico.  
+Some of my favorite hobbies include: 
+
+1.  Boxing  
```

Comment:
With markdown using 1. all places also works to get ordered list

Reply:
Would it be better to make it a list this way?
- Boxing
- Coding
- Hiking?